lib/presentation/screens/home_screen.dart

```diff
@@ -0,0 +1,92 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:ott_app/data/models/movie_model.dart';
+import 'package:ott_app/logic/cubit/movie_cubit.dart';
+import 'package:ott_app/logic/cubit/movie_state.dart';
+import 'package:ott_app/routes/app_router.gr.dart';
+
+class HomeScreen extends StatefulWidget {
+  const HomeScreen({super.key});
+
+  @override
+  State<HomeScreen> createState() => _HomeScreenState();
+}
+
+class _HomeScreenState extends State<HomeScreen> {
+  String imageUrl = 'https://image.tmdb.org/t/p/w185';
```

Comment:
@Aayushi12345 you can put this imageBaseUrl in constant file so that it will be use for detail and fav screen as well.